javatools/src/main/java/org/xvm/runtime/template/_native/lang/src/xRTCompiler.java

```diff
@@ -578,9 +578,9 @@ protected void log(Severity sev, String sMsg)
             }
 
         @Override
-        protected void abort(boolean fError)
+        protected void abort(boolean fError, final String msg)
```

Comment:
There is a great reason to start now, as it's part of the recommended code standard. Or at least - it's at least as great as starting now to rename packages to org.xtclang, moving lib directories into XDK, and updating the brace style and other aspects of the code to conform to the standards document we wanted. 

But it shouldn't be inconsistent per param, and it slipped through because I have it in my editor config.